app/pages/TransactionStatus.tsx

```diff
@@ -164,43 +164,38 @@ export function TransactionStatus({
 
   useEffect(
     function trackTransactionEvents() {
-      if (!isTracked) {
+      // Only track if we haven't tracked yet and have all required data
+      if (!isTracked && transactionStatus && completedAt) {
+        const bankName = getInstitutionNameByCode(
+          String(formMethods.watch("institution")),
+          supportedInstitutions,
+        );
+
+        const eventData = {
+          Amount: amount,
+          "Send token": token,
+          "Receive currency": currency,
+          "Recipient name": recipientName,
+          "Recipient bank": bankName,
+          "Noblocks balance": smartWalletBalance?.balances[token] || 0,
```

Comment:
we shouldn't track `recipient name`

Reply:
recipient name removed from the tracked data ✔️